otx/core/data/adapter/__init__.py

```diff
@@ -17,54 +17,80 @@
 # pylint: disable=too-many-return-statements
 import importlib
 
+from otx.algorithms.common.configs.training_base import TrainType
 from otx.api.entities.model_template import TaskType
 
 ADAPTERS = {
     TaskType.CLASSIFICATION: {
-        "module_name": "classification_dataset_adapter",
-        "class": "ClassificationDatasetAdapter",
+        "INCREMENTAL": {
```

Comment:
1. Doesn't classification selfsl task require adapter?
2. Can classification selfsl task get any adapter in this structure?